meta-lmp-bsp/dynamic-layers/meta-ti-bsp/recipes-support/ti-mfgtool-files/ti-mfgtool-files/flash.sh.in

```diff
@@ -131,40 +131,63 @@ fi
 [ -z "${boardType}" ] && boardType="gp"
 [ -z "${devID}" ] && devID="$(lsusb | grep 'Texas Instruments, Inc. AM62x' | awk '{print $6}')"
 
+vid=$(echo "${devID}" | cut -d ':' -f 1)
+pid=$(echo "${devID}" | cut -d ':' -f 2)
+
 ####
 # Flash software images
 ####
 
 # start USB DFU download mode and flash primary eMMC with provided WIC image
 
 echo "Load U-Boot via DFU..."
+dfu-suffix -c ${scriptDir}/tiboot3-am62x-${boardType}-evm.bin || dfuCheck=$?
+if [[ ${dfuCheck} -ne 0 ]]; then
+	echo "Adding DFU suffix signature to binaries"
+	dfu-suffix -v ${vid} -p ${pid} -a ${scriptDir}/tiboot3-am62x-${boardType}-evm.bin
```

Comment:
I would suggest that am62x be changed to %%MACHINE%% and converted when changed to flash.sh

Reply:
I was planning to do that in different PR when adding am64xx support